pkgs/development/libraries/libvirt/default.nix

```diff
@@ -147,14 +147,20 @@ stdenv.mkDerivation rec {
     sed -i '/virnetdaemontest/d' tests/meson.build
   '';
 
+  strictDeps = true;
+
   nativeBuildInputs = [
-    meson
+    meson # needs to be first
```

Comment:
cmake is overriding configurePhase. meson doesn't have to be first if `dontUseCmakeConfigure = true`